src/csharp/Grpc.Auth/Grpc.Auth.project.json

```diff
@@ -0,0 +1,8 @@
+{
+  "frameworks": {
+    "net45": { }
+  },
+  "runtimes": {
```

Comment:
Do you need the runtimes set? This is a library, so shouldn't need it - eventually it'll target netstandard1.5, right?